### PR TITLE
Change foo.com -> example.com

### PR DESCRIPTION
**Pull request by nancymic2**
- Change foo.com -> example.com
- .json extension to attestation filename

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Additional attestations may be added to individual APIs as scenarios arise.
 
 ### Validation and transparency framework
 
-Developers who submit an enrollment form are then sent a file that contains the attestations for the APIs they requested to use. To complete enrollment, the developer places the file in a site's public `.well-known` location. For example, if they enroll `example.com`, they need to place the attestation file at `https://foo.com/.well-known/privacy-sandbox-attestations`. 
+Developers who submit an enrollment form are then sent a file that contains the attestations for the APIs they requested to use. To complete enrollment, the developer places the file in a site's public `.well-known` location. For example, if they enroll `example.com`, they need to place the attestation file at `https://example.com/.well-known/privacy-sandbox-attestations`.
```
Comment (dmdabbs) [inline README.md:95]: Please confirm that the expected file URI is now 

`/.well-known/privacy-sandbox-attestations`

And is no longer 

`/.well-known/privacy-sandbox-attestations.json`

Comment (rowan-m) [reply README.md:95]: Raised #25 to address this!

**Review by rowan-m — CHANGES_REQUESTED** [review]
Consistent use of `example.com` LGTM, but move the scenarios to a separate file.

**Review by rowan-m — CHANGES_REQUESTED** [review]
Confirmed we should have the `.json` extension.

**Comment by nancymic2** [comment]
Also see b/312378501

**Review by rowan-m — APPROVED** [review]

**Pull request merged**